Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+HTTP1StateMachine.swift

```diff
@@ -29,16 +30,19 @@ extension HTTPConnectionPool {
 
         private(set) var requests: RequestQueue
         private(set) var lifecycleState: StateMachine.LifecycleState
+        private let retryConnectionEstablishment: Bool
```

Comment:
Can we add a comment here, why we have this property. We should note wherever we can, that this is a dull setting. :P